.github/workflows/main.yml

```diff
@@ -78,6 +78,7 @@ jobs:
           retention-days: 7
 
       - name: Deploy
+        continue-on-error: true
```

Comment:
@akaszynski I thought it was already the case because ligne 94:
`if: steps.deploy.outcome!='success' && github.event_name == 'push' && startsWith(github.ref, 'refs/tags')`
we check the outcome of the first deployment before to execute the second try?
Am I missing a point?